numba/ir_utils.py

```diff
@@ -1822,6 +1822,14 @@ def raise_on_unsupported_feature(func_ir, typemap):
     unsupported features.
     """
     gdb_calls = [] # accumulate calls to gdb/gdb_init
+    
+    for arg_name in func_ir.arg_names:
```

Comment:
Please add a comment above here about what the purpose of this block is. e.g. `# check for excessively large tuples`